ci/builders/linux_android_aot_engine.json

```diff
@@ -18,9 +18,6 @@
                 "device_type=none",
                 "os=Linux"
             ],
-            "gclient_custom_vars": {
```

Comment:
Thank you for the change! would it be possible to rename to gclient_variables instead?